openlr/router.cpp

```diff
@@ -396,6 +404,16 @@ double Router::GetPotential(Vertex const & u) const
   return potential;
 }
 
+bool Router::NearNextStage(Vertex const & u, double pi) const
+{
+  return u.m_stage < m_pivots.size() && pi < kEps;
+}
+
+bool Router::MayMoveToNextStage(Vertex const & u, double pi) const
```

Comment:
Why did you use a verb "may" instead of "can"?
If it reflects some usage context nuances, could you please leave a comment here?